client/public/index.html

```diff
@@ -4,31 +4,27 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
 
-    <!-- third-generation iPad with high-resolution Retina display: -->
     <link
-      rel="apple-touch-icon-precomposed"
-      sizes="144x144"
-      href="%PUBLIC_URL%/favicon144.png"
+      rel="shortcut icon"
+      href="%PUBLIC_URL%/favicon.ico"
+      type="image/x-icon"
     />
-
-    <!-- iPhone with high-resolution Retina display: -->
     <link
-      rel="apple-touch-icon-precomposed"
-      sizes="114x114"
-      href="%PUBLIC_URL%/favicon114.png"
+      rel="apple-touch-icon"
+      sizes="180x180"
+      href="%PUBLIC_URL%/apple-touch-icon.png"
     />
-
-    <!-- first- and second-generation iPad: -->
     <link
-      rel="apple-touch-icon-precomposed"
-      sizes="72x72"
-      href="%PUBLIC_URL%/favicon72.png"
+      rel="icon"
+      type="image/png"
+      sizes="32x32"
+      href="%PUBLIC_URL%/favicon-32x32.png"
     />
-
-    <!-- non-Retina iPhone, iPod Touch, and Android 2.1+ devices: -->
     <link
-      rel="apple-touch-icon-precomposed"
-      href="%PUBLIC_URL%/favicon57.png"
+      rel="icon"
+      type="image/png"
+      sizes="16x16"
+      href="%PUBLIC_URL%/favicon-16x16.png"
     />
```

Comment:
I believe we also need:

```html
<!-- For Chrome on Android -->
<link rel="icon" href="/favicon-192x192.png" sizes="192x192" type="image/png" />
```

https://realfavicongenerator.net/blog/android-chrome-and-its-favicon/